index.js

```diff
@@ -1,12 +1,18 @@
 import { AppRegistry, YellowBox } from 'react-native'
 import App from './src/modules/app/components/App'
+import { Buffer } from 'buffer'
+import { URL } from 'whatwg-url'
 import 'moment/locale/de' // fixme
 
 // @see: https://github.com/facebook/react-native/issues/9599
 if (typeof global.self === 'undefined') {
   global.self = global
 }
 
+// @see https://github.com/facebook/react-native/issues/16434
+global.Buffer = Buffer
+global.URL = URL
```

Comment:
Are you sure you need to override the current one in the global scope?
As far as I understand the usual URL implementation is missing. So can you just import URL in `createNavigateToIntegreatUrl.js` and use the imported one?

I think we do not need to override this one.

Reply:
Maybe just importing URL works, but afaik buffer has to be set globally, because URL uses it internally.